app/src/main/java/com/x8bit/bitwarden/data/platform/manager/NetworkConfigManagerImpl.kt

```diff
@@ -41,6 +43,7 @@ class NetworkConfigManagerImpl(
             .environmentStateFlow
             .onEach { environment ->
                 baseUrlInterceptors.environment = environment
+                serverConfigRepository.getServerConfig(forceRefresh = true)
```

Comment:
During startup, this can change really fast. I wonder if we should debounce this by maybe 500 milliseconds, thoughts?